### ISSUE TITLE
I tested this mediawiki-blockstack-sso on bitnami XAMPP on my macOS , not work.

### ISSUE THREAD

**Issue by Deutsch4534** [saul-avikar/mediawiki-blockstack-sso#1]
@saul-avikar @Aranad 
my computer: macOS 10.14.5(Mojave),and download the xampp .

**Here are my steps:**

1. Downloaded the  from mediawiki from https://bitnami.com/stack/mediawiki/installer, and install it successfully and I can login on 192.168.64.2
2.  Downloaded this mediawiki-blockstack-sso repository and unzip it ,renamed it to BlockstackSso in  extensions/ folder. 

![屏幕快照 2019-06-11 下午9 19 20](https://user-images.githubusercontent.com/12605959/59275720-786afc00-8c8f-11e9-8853-69ae7ea15844.png)

3. Add the following code at the bottom of your LocalSettings.php:
wfLoadExtension( 'BlockstackSso' );

![屏幕快照 2019-06-11 下午9 18 32](https://user-images.githubusercontent.com/12605959/59275773-8f115300-8c8f-11e9-8057-646bee513af8.png)
![屏幕快照 2019-06-11 下午9 18 44](https://user-images.githubusercontent.com/12605959/59275775-8f115300-8c8f-11e9-9c71-804e547287cd.png)

4. restarted the application , and go to 192.168.64.2, clicked the login with blockstack . cannot redirect to blockstack.Here are the console errors:

![屏幕快照 2019-06-11 下午9 17 16](https://user-images.githubusercontent.com/12605959/59275925-e0214700-8c8f-11e9-8eba-22cfb5000d85.png)

5. But ,it works well with bitnami login 

Username:user
Password:bitnami1

![屏幕快照 2019-06-11 下午9 30 15](https://user-images.githubusercontent.com/12605959/59276116-44dca180-8c90-11e9-9d4c-c6dcdf7637bf.png)

**Comment by saul-jb** [comment]
Have you installed the blocktack-sso lib (https://github.com/saul-avikar/blockstack-sso) as per the installation instructions?

> Note that this extension depends on the Blockstack SSO single-signon for PHP apps, and expects it to be situated in the base directory of the extension and named BlockstackCommon (case-sensitive). If you're running many different sites that all allow signing in with Blockstack, then it's best to keep just one copy of the common dependency in your environment and symlink to them from each site that needs it.

Another (unrelated) issue with your setup is that it is a local install and is not reachable by the Blockstack service to fetch the manifest page and Blockstack will reject the sign-in request.

**Comment by Deutsch4534** [comment]
@saul-avikar  Yes, please check the step 2 and step 3 to view what I did.And I followed this tutorial https://www.mediawiki.org/wiki/Extension:BlockstackSSO

server location:
nfs://192.168.64.2/opt/bitnami/apps/mediawiki/htdocs/extensions/BlockstackSso

file directory:
bitnami⁩ ▸ ⁨apps⁩ ▸ ⁨mediawiki⁩ ▸ ⁨htdocs⁩ ▸ ⁨extensions

As your tutorial said , the folder should be named BlockstackSso, but you just said named BlockstackCommon

I don't have manifest file in that bitnami/apps/mediawiki/htdocs/extensions/BlockstackSso directory.
Can you upload the all the files that we need to installed?And write a new tutorial?
Are you mean that i need to  install https://github.com/saul-avikar/blockstack-sso instead of https://github.com/saul-avikar/mediawiki-blockstack-sso to the mediawiki⁩ ▸ ⁨htdocs⁩ ▸ ⁨extensions folder?

**Comment by saul-jb** [comment]
### Directory Structure
You need both the blockstack-sso library and the mediawiki-blockstack-sso extension, install the blockstack-sso lib (https://github.com/saul-avikar/blockstack-sso) inside "BlockstackSso" directory and rename it to "BlockstackCommon" so your directory structure should look something like this:

- ⁨mediawiki
  - ⁨extensions
    - **BlockstackSso** (https://github.com/saul-avikar/mediawiki-blockstack-sso)
      -  auth
      -  i18n
      - modules
      -  styles
      - **BlockstackCommon** (https://github.com/saul-avikar/blockstack-sso)
----
### Manifest
The extension should handle the manifest itself so you don't need to worry about it, **but** it needs to be accessible from the internet and over SSL (https) - 192.168.64.2 is a local address the Blockstack service cannot access that. So you need to setup an external IP address, map a domain name to it and configure SSL - best to use a hosting service or server for this.
